.gitattributes

```diff
@@ -0,0 +1,2 @@
+*       text=auto
+*.sh    text        eol=lf
```

Comment:
Prevents those on Windows converting the Docker scripts to CLRF, which breaks Docker containers.